oxide-auth-poem/Readme.md

```diff
@@ -0,0 +1,20 @@
+# oxide-auth-poem
+
+Integrates `oxide-auth` with the [`poem`] web server library.
+
+## Additional
+
+[![Crates.io Status](https://img.shields.io/crates/v/oxide-auth-iron.svg)](https://crates.io/crates/oxide-auth-iron)
+[![Docs.rs Status](https://docs.rs/oxide-auth-iron/badge.svg)](https://docs.rs/oxide-auth-iron/)
+[![License](https://img.shields.io/badge/license-MIT-blue.svg)](https://raw.githubusercontent.com/HeroicKatora/oxide-auth/dev-v0.4.0/docs/LICENSE-MIT)
+[![License](https://img.shields.io/badge/license-Apache-blue.svg)](https://raw.githubusercontent.com/HeroicKatora/oxide-auth/dev-v0.4.0/docs/LICENSE-APACHE)
+[![CI Status](https://api.cirrus-ci.com/github/HeroicKatora/oxide-auth.svg)](https://cirrus-ci.com/github/HeroicKatora/oxide-auth)
+
+Licensed under either of
+* MIT license ([LICENSE-MIT] or http://opensource.org/licenses/MIT)
+* Apache License, Version 2.0 ([LICENSE-APACHE] or http://www.apache.org/licenses/LICENSE-2.0)
+  at your option.
+
+[`iron`]: https://crates.io/crates/poem
```

Comment:
```suggestion
[`poem`]: https://crates.io/crates/poem
```

This link label should also be adjusted (was adjusted in the text above).

Reply:
oops, I was 99% sure I edited this properly...
I probably messed up the git commit :p 

thanks for catching this!